tests/queue/QueueSummaryNormalizerTest.php

```
@@ -185,7 +185,7 @@ public function testFromPanelDataReturnsRecordsInOriginalOrder(): void
             ],
         );
 
-        $classes = array_map(static fn($r) => $r->jobClass, $summary->records);
+        $classes = array_map(static fn(JobRecord $r): string => $r->jobClass, $summary->records);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $r. Configured minimum length is 3. (undefined)

(ShortVariable)